lib/reek/context/attribute_context.rb

```diff
@@ -15,6 +15,15 @@ def full_comment
         send_expression.full_comment || ''
       end
 
+      def instance_method?
+        true
+      end
```

Comment:
Detected MethodEnvy Smell! See what I did there? I just invented a new smell type, cause this is how we roll!
In other news, I have a problem with the gazillion of overwritten methods. I know that we're using the template pattern here but it feels wrong since we're talking about predicate methods here. And we **know** what they return.

Long story short: Feels like we're mixing up "code" with "data" here.
How about using a map we reference via class instance variable in each context like this:

```Ruby
class AttributeContext < ...
  @yolo = {  # Trying to come up with a good name here. "method_properties"?
    instance_method: true,
    # ....
  }
```

wdyt?